compiler/src/dotty/tools/dotc/core/TypeComparer.scala

```diff
@@ -1641,6 +1643,18 @@ class TypeComparer(initctx: Context) extends ConstraintHandling[AbsentContext] {
         NoType
     }
 
+  private[this] def andTypeGen(tp1: Type, tp2: Type, op: (Type, Type) => Type
+    , original: (Type, Type) => Type = _ & _, isErased: Boolean = ctx.erasedTypes): Type = trace(s"glb(${tp1.show}, ${tp2.show})", subtyping, show = true) {
```

Comment:
Put commas at the end o lines, not at the beginning.

Reply:
Ok, updated